source/server/configuration_impl.cc

```diff
@@ -127,11 +127,12 @@ InitialImpl::InitialImpl(const envoy::config::bootstrap::v2::Bootstrap& bootstra
     flags_path_ = bootstrap.flags_path();
   }
 
-  if (bootstrap.has_runtime()) {
-    runtime_ = std::make_unique<RuntimeImpl>();
-    runtime_->symlink_root_ = bootstrap.runtime().symlink_root();
-    runtime_->subdirectory_ = bootstrap.runtime().subdirectory();
-    runtime_->override_subdirectory_ = bootstrap.runtime().override_subdirectory();
+  base_runtime_ = bootstrap.runtime().base();
```

Comment:
Can you check the current info startup output for runtime and maybe have some printing on if runtime is configured, whether a base runtime was specified, whether a disk runtime was specified or not, etc. I think this would help with config errors now that disk is not a required field?